staging/src/k8s.io/kubectl/pkg/kuberc/marshal.go

```diff
@@ -87,14 +93,28 @@ func decodePreference(kubercFile string) (*config.Preference, error) {
 			continue
 		}
 
+		// check version priority
+		if gvk.GroupVersion() == v1alpha1.SchemeGroupVersion {
+			olderVersions[v1alpha1.SchemeGroupVersion.String()] = preferences
```

Comment:
if multiple v1alpha1s are encountered, this will make the last one encountered win, instead of the first, as before, right?

Reply:
Good catch, I'll add a test-case and will fix it.